src/app/basics/getting-started/record-your-cypress-tests/page.md

```diff
@@ -49,6 +49,12 @@ bun install @replayio/cypress -D
 {% /tab %}
```

Comment:
Looks like Bun got missed in the Cypress page, cc @jonbell-lot23

Reply:
Thanks, fixed here: https://github.com/replayio/docs/pull/190